apps/modernization-ui/src/apps/page-builder/page/management/preview/PreviewPage.tsx

```diff
@@ -29,14 +29,15 @@ const PreviewPage = () => {
 
 const PreviewPageContent = () => {
     const { page, selected } = usePageManagement();
+    const { pageId } = useParams();
```

Comment:
Suggestion: The `pageId` should be available on the `page` object returned from the `usePageManagement()`